.circleci/config.yml

```diff
@@ -959,7 +959,8 @@ jobs:
       - checkout
       - install-dependencies:
           install_xcbeautify: false
-          install_mint: true
+          # Mint 0.17.5 is required here as newer versions of Mint don't support macOS 12.
```

Comment:
We need to force installing mint 17.5, since mint 18+ don't support macOS 12, which is required for iOS 13/14.

See https://github.com/RevenueCat/purchases-ios/pull/5256 for more details

Reply:
Once we rebase main this will not be needed then?